### ISSUE TITLE
"contract object doesn't have address set yet" appeared during the monitor work

### ISSUE THREAD

**Issue by akolotov** [poanetwork/tokenbridge#79]
the xDai bridge monitor reported about an unhandled promise:
```
(node:31982) UnhandledPromiseRejectionWarning: Error: This contract object doesn't have address set yet, please set an address first.
    at Contract._generateEventOptions (/home/bridgemon/bridge-monitor-nodejs/node_modules/web3-eth-contract/src/index.js:538:15)
    at Contract.getPastEvents (/home/bridgemon/bridge-monitor-nodejs/node_modules/web3-eth-contract/src/index.js:649:49)
    at main (/home/bridgemon/bridge-monitor-nodejs/stuckTransfers.js:82:49)
    at checkWorker3 (/home/bridgemon/bridge-monitor-nodejs/checkWorker3.js:9:29)
    at Object.<anonymous> (/home/bridgemon/bridge-monitor-nodejs/checkWorker3.js:23:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
(node:31982) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
```

The config file:
```
HOME_RPC_URL=https://dai.poa.network
FOREIGN_RPC_URL=<url to the eth mainnet>
HOME_BRIDGE_ADDRESS=0x7301CFA0e1756B71869E93d4e4Dca5c7d0eb0AA6
FOREIGN_BRIDGE_ADDRESS=0x4aa42145Aa6Ebf72e164C9bBC74fbD3788045016
HOME_DEPLOYMENT_BLOCK=759
FOREIGN_DEPLOYMENT_BLOCK=6478417
GAS_PRICE_SPEED_TYPE=standard
GAS_LIMIT=300000
GAS_PRICE_FALLBACK=21
LEFT_TX_THRESHOLD=100
```

**Comment by patitonar** [comment]
From this log `at checkWorker3 (/home/bridgemon/bridge-monitor-nodejs/checkWorker3.js:9:29)` it looks the error comes from running `checkWorker3` but I don't think the monitor should be running that script for xDai bridge since documentation mentions:
```
 # only applicable for bridge-rust-v1-native-to-erc
node checkWorker3.js 
```
Actually, since that script has no purpose for v2 contracts, I think we should remove it to avoid this kind of confusions.

**Comment by akolotov** [comment]
Addressed in #75
